triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -789,9 +800,17 @@ object Runner extends StrictLogging {
     */
   val maxParallelSubmissionsPerTrigger = 8
   val maxTriesWhenOverloaded = 6
+  val minTriesBackoff: FiniteDuration = 250.milliseconds
+  val maxTriesBackoff: FiniteDuration = 8.seconds
 
-  private def overloadedRetryDelay(afterTries: Int): FiniteDuration =
-    (250 * (1 << (afterTries - 1))).milliseconds
+  /** Percentage jitter (expressed as a value in the unit interval [0, 1]) to add to
+    * each backoff duration
+    */
+  val jitterTries = 0.2
+
+  private val submissionRestartSettings =
+    RestartSettings(minTriesBackoff, maxTriesBackoff, jitterTries)
+      .withMaxRestarts(maxTriesWhenOverloaded, maxTriesBackoff)
```

Comment:
`withMaxRestarts(maxTriesWhenOverloaded, 15750ms)` should be capturing the original number of retries over the entire `retrying` time window